subsys/bluetooth/services/fast_pair/fp_keys.c

```diff
@@ -533,5 +534,37 @@ static int fp_keys_uninit(void)
 	return 0;
 }
 
+int fp_keys_bond_save(const struct bt_conn *conn)
+{
+	if (!IS_ENABLED(CONFIG_BT_FAST_PAIR_BOND_MANAGER)) {
+		__ASSERT_NO_MSG(false);
+		return -ENOTSUP;
+	}
+	__ASSERT_NO_MSG(bt_fast_pair_is_ready());
+
+	struct fp_procedure *proc = &fp_procedures[bt_conn_index(conn)];
+	const struct fp_account_key *account_key;
```

Comment:
the build system has issues detecting it as uninitialized variable, you can initialize it to NULL just to suppress this warning